onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -14,6 +15,210 @@
 from .registry import IntegerOpsRegistry, QLinearOpsRegistry
 
 
+class ExecutionProvider(Enum):
+    CPU = 1
+    TRT = 2
+    NNAPI = 3
+    SNE = 4
```

Comment:
`enum.auto()` if we don't care about the numbers. [`@enum.unique` ](https://docs.python.org/3/library/enum.html#ensuring-unique-enumeration-values) if manually assigned